torchchat/utils/quantize.py

```diff
@@ -886,10 +784,10 @@ def quantized_model(self) -> nn.Module:
 # class references
 quantizer_class_dict = {
     "embedding": EmbeddingOnlyQuantHandler,
-    "linear:int8": WeightOnlyInt8QuantHandler,
     "precision": PrecisionHandler,
     "executor": ExecutorHandler,
     "linear:int4": Int4WeightOnlyQuantizer,
+    "linear:int8": int8_weight_only,
```

Comment:
Do we need this?

Reply:
we can probably use None for now, and remove this later

Reply:
We check for int8_weight_only and finished check before it looks at the table I think

@vmpuri can you check?